notifications.yaml

```diff
@@ -0,0 +1,242 @@
+      title:
+        type: string
+        description: |
+          A short, summary of the problem type. Written in english and readable
+          for engineers (usually not suited for non technical stakeholders and
+          not localized); example: Service Unavailable
+      status:
+        type: integer
+        format: int32
+        description: |
+          The HTTP status code generated by the origin server for this occurrence
+          of the problem.
+        minimum: 100
+        maximum: 600
+        exclusiveMaximum: true
+        example: 503
+      detail:
+        type: string
+        description: |
+          A human readable explanation specific to this occurrence of the
+          problem.
+        example: Connection to database timed out
+      instance:
+        type: string
+        format: uri
+        description: |-
+          An absolute URI that identifies the specific occurrence of the problem.
+          It may or may not yield further information if dereferenced.
+  MessageContent:
+    title: MessageContent
+    type: object
+    properties:
+      body_short:
+        type: string
+        description: 'An abstract of the message, in plain text. The content of this field will be delivered to channels that support a limited amount of characters (e.g. SMS, mobile notifications, etc...).'
+        minLength: 3
+        maxLength: 100
+      body_long:
+        type: string
+        description: 'A long version of the message, in plain text. The content of this field will be delivered to channels that don''t have any limit in terms of content size (e.g. email, etc...). The long version of the message is optional and should be provided only in cases when a longer message (compared to the short version provided in body_short) gives more information to the user. This field should not be a duplicate of body_short.'
+        minLength: 100
+        maxLength: 100000
+    required:
+      - body_short
+  NewMessage:
+    title: NewMessage
+    type: object
+    properties:
+      dry_run:
+        type: boolean
+        description: 'This parameter, when set to true, allows developers to test a request without actually sending a message. The submitted message won''t be delivered to the end user but only checked for validity: "400 BAD REQUEST" is returned with a payload that describes the error. If the message can be accepted for delivery then "200 OK" is returned. The default value is false.'
+      time_to_live:
+        $ref: '#/definitions/TimeToLive'
+      content:
+        $ref: '#/definitions/MessageContent'
+    required:
+      - content
+  MessageStatus:
+    title: MessageStatus
+    type: object
+    properties:
+      created_at:
+        type: string
+        format: date-time
+        description: 'The date-time the message has been created (delivered to the user''s inbox). The format is defined by RFC 3339, for example 2015-05-28T14:07:17Z.'
+      read_at:
```

Comment:
This should be optional (can be null in case the user has not yet read the message), not sure how you would define that in the specs, at least we should explain it in the description.

Also, we should explain that this has set-once behavior.